### ISSUE TITLE
Regression in 1.6.5 when mocking an interface on PHP 8.2

### ISSUE THREAD

**Issue by GrahamCampbell** [mockery/mockery#1326]
### Mockery Version

1.6.5

### PHP Version

PHP 8.2

### Issue Description

* Mockery 1.6.4 is working fine, on both PHP 8.1 and 8.2.
* Mockery 1.6.5 works fine on PHP 8.1, but is broken on PHP 8.2 when trying to mock `Symfony\Contracts\HttpClient\HttpClientInterface`.

### Steps to Reproduce

```
Mockery::mock(Symfony\Contracts\HttpClient\HttpClientInterface::class)
```

### Expected Behavior

It not to crash.

### Actual Behavior

```
Fatal error: Declaration of Mockery_9_Symfony_Contracts_HttpClient_HttpClientInterface::stream(Traversable|array $responses, ?float $timeout = null): Symfony\Contracts\HttpClient\ResponseStreamInterface must be compatible with Symfony\Contracts\HttpClient\HttpClientInterface::stream(Symfony\Contracts\HttpClient\ResponseInterface|Traversable|array $responses, ?float $timeout = null): Symfony\Contracts\HttpClient\ResponseStreamInterface in /Users/graham/GitHub/Voxie/libs/event-publisher-php/vendor/mockery/mockery/library/Mockery/Loader/EvalLoader.php(24) : eval()'d code on line 978
```

### Additional Information

```
mockery/mockery                    1.6.5
symfony/http-client                v6.3.2
symfony/http-client-contracts      v3.3.0
```

**Comment by ghostwriter** [comment]
Hey @GrahamCampbell,

Thanks for reporting this issue, we've addressed this issue via #1324 ( `composer require mockery/mockery:1.6.x-dev`)

I'll create a tag shortly.

**Comment by ghostwriter** [comment]
Fixed via #1324 

Tagged: https://github.com/mockery/mockery/releases/tag/1.6.6
